Buildings/Examples/VAVReheat/Guideline36.mo

```diff
@@ -26,39 +26,39 @@ model Guideline36
     AFlo=AFloCor,
     final samplePeriod=samplePeriod,
     VDisSetMin_flow=max(1.5*VCorOA_flow_nominal, 0.15*mCor_flow_nominal/1.2),
-    VDisHeaSetMax_flow=ratVFloHea*VCorOA_flow_nominal)
+    VDisHeaSetMax_flow=ratVFloHea*mCor_flow_nominal/1.2)
     "Controller for terminal unit corridor"
     annotation (Placement(transformation(extent={{530,84},{550,104}})));
   Buildings.Controls.OBC.ASHRAE.G36_PR1.TerminalUnits.Controller conVAVSou(
     V_flow_nominal=mSou_flow_nominal/1.2,
     AFlo=AFloSou,
     final samplePeriod=samplePeriod,
     VDisSetMin_flow=max(1.5*VSouOA_flow_nominal, 0.15*mSou_flow_nominal/1.2),
-    VDisHeaSetMax_flow=ratVFloHea*VSouOA_flow_nominal)
+    VDisHeaSetMax_flow=ratVFloHea*mSou_flow_nominal/1.2)
     "Controller for terminal unit south"
     annotation (Placement(transformation(extent={{702,84},{722,104}})));
   Buildings.Controls.OBC.ASHRAE.G36_PR1.TerminalUnits.Controller conVAVEas(
     V_flow_nominal=mEas_flow_nominal/1.2,
     AFlo=AFloEas,
     final samplePeriod=samplePeriod,
     VDisSetMin_flow=max(1.5*VEasOA_flow_nominal, 0.15*mEas_flow_nominal/1.2),
-    VDisHeaSetMax_flow=ratVFloHea*VEasOA_flow_nominal)
+    VDisHeaSetMax_flow=ratVFloHea*mEas_flow_nominal/1.2)
     "Controller for terminal unit east"
     annotation (Placement(transformation(extent={{880,84},{900,104}})));
   Buildings.Controls.OBC.ASHRAE.G36_PR1.TerminalUnits.Controller conVAVNor(
     V_flow_nominal=mNor_flow_nominal/1.2,
     AFlo=AFloNor,
     final samplePeriod=samplePeriod,
     VDisSetMin_flow=max(1.5*VNorOA_flow_nominal, 0.15*mNor_flow_nominal/1.2),
-    VDisHeaSetMax_flow=ratVFloHea*VNorOA_flow_nominal)
+    VDisHeaSetMax_flow=ratVFloHea*mNor_flow_nominal/1.2)
     "Controller for terminal unit north"
     annotation (Placement(transformation(extent={{1038,84},{1058,104}})));
   Buildings.Controls.OBC.ASHRAE.G36_PR1.TerminalUnits.Controller conVAVWes(
     V_flow_nominal=mWes_flow_nominal/1.2,
     AFlo=AFloWes,
     final samplePeriod=samplePeriod,
     VDisSetMin_flow=max(1.5*VWesOA_flow_nominal, 0.15*mWes_flow_nominal/1.2),
-    VDisHeaSetMax_flow=ratVFloHea*VWesOA_flow_nominal)
+    VDisHeaSetMax_flow=ratVFloHea*mWes_flow_nominal/1.2)
```

Comment:
These were the issues that caused the discrepancies between `master` and this branch. These parameters were set to `ratVFloHea*VWesOA_flow_nominal`, when originally, they were set to `0.3*V_flow_nominal`. I think the intention of that line change was to reuse the `ratVFloHea` variable, but the wrong flow rate was used (outside air instead of mixed), which caused the controller to limit the amount of air going into the spaces.

Reply:
@mwetter I'll let you take another look at the test results, but this does fix the main gap you pointed out. I can update the reference results in the morning.